src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Pages/Counter.razor

```diff
@@ -1,5 +1,7 @@
 ﻿@page "/counter"
 
+<PageTitle>Counter</PageTitle>
+
```

Comment:
This is a very minor detail but I wonder if the readability would be improved by putting pagetitle in the top block with the directives. That is, removing the blank line 2.

@danroth27 What do you think will be most obvious to total newcomers?

I’m totally happy for it to stay as-is in this PR if others don’t think it would be an improvement to remove that blank line.

Reply:
Do you mean eliminate the space after the `@page` directive in this case? I think most of the developers will be adding that separator line back as soon as they see those two together. So I'm fine with this, but let's see what @danroth27 thinks.

Reply:
I prefer to leave the blank line. I think most folks will expect some white space separation between the directives at the top and any markup.

Reply:
For me it’s not quite markup, at least in the sense of the role it plays. 

But it was just an offhanded thought. The code layout as-is will probably seem normal soon. Keeping it like this is fine with me.